content/docs/insights/get-started/begin.md

```diff
@@ -0,0 +1,191 @@
+---
+title_tag: Before You Begin | Pulumi Insights
+title: Before you begin
+h1: "Pulumi Insights: Before you begin"
+meta_desc: This page provides an overview on how to get started with Pulumi Insights Accounts.
+weight: 2
+menu:
+  insights:
+    parent: insights-get-started
+    identifier: insights-get-started-begin
+    weight: 2
+---
+
+## Before you begin
+
+First, let's run through a few prerequisites and quick steps to ensure you ready to create your first Account Discovery scan.
+
+- Ensure you’re an admin of your Pulumi organization.
```

Comment:
We will be removing this requirement soon. Members will be allowed to create accounts so long as their org settings allow it

Reply:
I'll keep it for now, and remove when this launches.